omx/gstomxaacdec.c

```diff
@@ -96,70 +98,113 @@ gst_omx_aac_dec_set_format (GstOMXAudioDec * dec, GstOMXPort * port,
   const gchar *stream_format;
 
   gst_omx_port_get_port_definition (port, &port_def);
-  port_def.format.audio.eEncoding = OMX_AUDIO_CodingAAC;
+
+  if (port->index == OMX_AACDEC_INPUT_PORT) {
+    port_def.nPortIndex = port->index;
+    port_def.eDir = OMX_DirInput;
+    port_def.nBufferCountActual = 1;
+    port_def.nBufferCountMin = 1;
+    port_def.nBufferSize = 4096;
```

Comment:
We could create a definition for this number as well

Reply:
66cee90a7d9904dfc8155591da324492d4f520ef